src/client/client.tsx

```diff
@@ -0,0 +1,7 @@
+import EventsJSON from "../data/events.json";
+
+export function fetchEvent(id: Number): Promise<Event> {
+  return new Promise(() => EventsJSON.find((e) => e["id"] === id));
```

Comment:
For this `Promise` object do you need to also include `resolve` and `reject` params?

Reply:
Yess you were right